### DISCUSSION TITLE
Force a resource property to be added, even if CDK doesn't know about it?

### DISCUSSION THREAD

**Discussion by greensmith** [aws/aws-cdk#32183] [Q&A]
TL;DR  is there a way to force a property on a cfn resource, even if CDK doesn't know it exists?

Hi,

I've tried searching for a possible solution to this, but it is hidden under the mountain of resources suggesting addPropertyOverride / addOverride 

Neither of those options seem to be able to actually "force" a property that CDK is unaware of.

In my particular case I am trying to apply SNSNotificationArn to CfnApplication for Application Insights. 

The property has been recently added to the CloudFormation resource but as of the latest release, 2.167.1, cdk isn't aware of this property 

I presumed due to the usual lag between cloudformation updates and cdk updates someone somewhere would have found a way to do this already?

Thanks
